x-pack/plugins/apm/server/routes/agent_keys/route.ts

```diff
@@ -20,7 +19,11 @@ const agentKeysRoute = createApmServerRoute({
   endpoint: 'GET /internal/apm/agent_keys',
   options: { tags: ['access:apm'] },
 
-  handler: async (resources) => {
+  handler: async (
+    resources
+  ): Promise<{
+    agentKeys: Array<import('./../../../../security/common/index').ApiKey>;
```

Comment:
it can be just me 🙈  but I'm not familiar with this syntax.  it is the same as the following?
```
import { ApiKey } from('./../../../../security/common/index');
...
...
agentKeys: Array<ApiKey>;
```
Does it have any other impact?

Reply:
pretty much! it's a dynamic import statement, the TypeScript way. it's essentially the same, just inlined with the type. it is automatically generated by TypeScript.